include/openssl/aes.h

```diff
@@ -141,6 +141,24 @@ OPENSSL_EXPORT void AES_ofb128_encrypt(const uint8_t *in, uint8_t *out,
                                        size_t len, const AES_KEY *key,
                                        uint8_t *ivec, int *num);
 
+// AES_cfb1_encrypt encrypts (or decrypts, if |enc| == |AES_DECRYPT|) |len|
```

Comment:
NP: I know the documentation was just copied from `AES_cfb128_encrypt`, but it's silent about several parameters for which usage is ambiguous. (I'm not familiar with the CFB cipher mode.)